PyDSS/dssInstance.py

```diff
@@ -402,8 +405,9 @@ def RunStep(self, step, updateObjects=None):
 
         if self._settings.helics.co_simulation_mode:
             self._HI.updateHelicsPublications()
-            self._increment_flag, helics_time = self._HI.request_time_increment()
+            self._increment_flag, helics_time = self._HI.request_time_increment(step=step)
 
+        # TODO DT: This is a potential problem for Gemini because it was returning the results dict.
```

Comment:
The RunStep method should return `(has_converged, {})` to include results.

Reply:
@AadilLatif It is not possible to return results from `RunStep` because the results are now collected in `RunSimulation` instead. Do we need to discuss this?